source/agora/utils/Test.d

```diff
@@ -344,3 +344,248 @@ unittest
+            Seed([58, 158, 102, 182, 77, 155, 91, 79, 35, 70, 139, 238, 106, 199, 96, 28, 107, 204, 167, 182, 51, 166, 184, 202, 21, 201, 192, 140, 78, 150, 126, 45]));
+        static immutable X = KeyPair(
+            PublicKey([47, 169, 193, 230, 87, 215, 19, 206, 13, 40, 52, 64, 141, 70, 89, 225, 195, 22, 206, 153, 195, 78, 122, 46, 216, 255, 17, 63, 160, 90, 2, 176]),
+            SecretKey([130, 11, 231, 41, 24, 62, 67, 165, 232, 188, 144, 20, 154, 27, 208, 239, 130, 169, 235, 54, 148, 230, 155, 207, 87, 48, 204, 114, 144, 91, 245, 88, 47, 169, 193, 230, 87, 215, 19, 206, 13, 40, 52, 64, 141, 70, 89, 225, 195, 22, 206, 153, 195, 78, 122, 46, 216, 255, 17, 63, 160, 90, 2, 176]),
+            Seed([130, 11, 231, 41, 24, 62, 67, 165, 232, 188, 144, 20, 154, 27, 208, 239, 130, 169, 235, 54, 148, 230, 155, 207, 87, 48, 204, 114, 144, 91, 245, 88]));
+        static immutable Y = KeyPair(
+            PublicKey([49, 169, 14, 77, 77, 152, 159, 177, 120, 130, 224, 30, 67, 70, 27, 88, 38, 230, 171, 161, 76, 233, 119, 95, 46, 186, 48, 75, 130, 51, 49, 121]),
+            SecretKey([234, 172, 224, 221, 152, 224, 1, 88, 53, 171, 161, 242, 240, 158, 113, 0, 80, 100, 33, 8, 210, 67, 190, 164, 146, 97, 126, 207, 151, 211, 226, 178, 49, 169, 14, 77, 77, 152, 159, 177, 120, 130, 224, 30, 67, 70, 27, 88, 38, 230, 171, 161, 76, 233, 119, 95, 46, 186, 48, 75, 130, 51, 49, 121]),
+            Seed([234, 172, 224, 221, 152, 224, 1, 88, 53, 171, 161, 242, 240, 158, 113, 0, 80, 100, 33, 8, 210, 67, 190, 164, 146, 97, 126, 207, 151, 211, 226, 178]));
+        static immutable Z = KeyPair(
+            PublicKey([51, 164, 176, 208, 41, 194, 54, 100, 117, 27, 62, 181, 147, 117, 202, 86, 11, 172, 24, 197, 191, 26, 171, 79, 107, 212, 224, 0, 65, 133, 64, 147]),
+            SecretKey([74, 77, 172, 53, 11, 228, 71, 106, 164, 11, 70, 212, 207, 242, 201, 199, 225, 255, 226, 255, 109, 169, 39, 109, 189, 241, 103, 219, 52, 24, 10, 206, 51, 164, 176, 208, 41, 194, 54, 100, 117, 27, 62, 181, 147, 117, 202, 86, 11, 172, 24, 197, 191, 26, 171, 79, 107, 212, 224, 0, 65, 133, 64, 147]),
+            Seed([74, 77, 172, 53, 11, 228, 71, 106, 164, 11, 70, 212, 207, 242, 201, 199, 225, 255, 226, 255, 109, 169, 39, 109, 189, 241, 103, 219, 52, 24, 10, 206]));
+    }
+}
+
+/// Generate all the well-kwnon values, disabled but kept here for documentation
+/// Note that we generate the binary data directly to limit CTFE overhead
+version (none) unittest
+{
+    import std.stdio;
+
+    Seed[size_t] kps;
+    string name = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"; // Writefln is picky
+    while (kps.length < 26)
+    {
+        auto tmp = KeyPair.random();
+        const addr = tmp.address.toString();
+        /* Addresses always start with G{A,B,C,D}
+         * We pick the addresses that start with GA, then match our expected
+         * char, and are followed by a `2`, so that we have an 'end marker'
+         * in case we want more letters (e.g. AA or AAA).
```

Comment:
yep. And if we need more than 1000 we can just resort back to using random keys. Nice work.